<?xml version="1.0" ?>
<!--
Copyright 2020 Google LLC.

This work is licensed under the Creative Commons Attribution 4.0
International License. To view a copy of this license, visit
http://creativecommons.org/licenses/by/4.0/ or send a letter
to Creative Commons, PO Box 1866, Mountain View, CA 94042, USA.
-->
<sdf version="1.6">
  <model name="bowl2AV">
    <link name="link_0">
      <visual name="visual">
        <geometry>
          <mesh>
            <uri>model://bowl2AV/meshes/model.obj</uri>
          </mesh>
        </geometry>
        <material>
          <ambient>1 0 0 1</ambient>
          <diffuse>1 0 0 1</diffuse>
          <specular>0.1 0.1 0.1 1</specular>
          <emissive>0 0 0 0</emissive>
        </material>
      </visual>
      <collision name="collision_0">
        <geometry>
          <mesh>
            <uri>model://bowl2AV/meshes/model.obj</uri>
          </mesh>
        </geometry>
        <surface>
          <contact>
            <ode>
              <max_vel>0.1</max_vel>
              <min_depth>0.001</min_depth>
              <soft_erp>0.5</soft_erp>
              <soft_cfm>0.001</soft_cfm>
            </ode>
          </contact>
          <friction>
            <ode>
              <mu>10000.0</mu>
              <mu2>10000.0</mu2>
            </ode>
          </friction>
        </surface>
      </collision>
    </link>
  </model>
</sdf>
